src/cephadm/cephadm

```diff
@@ -1434,7 +1473,7 @@ def _filter_last_local_ceph_image(out):
 
 
 def write_tmp(s, uid, gid):
-    # type: (str, int, int) -> Any
+    # type: (str, int, int) -> IO[Any]
```

Comment:
```suggestion
    # type: (str, int, int) -> TextIO
```

Reply:
ack

Reply:
Apparently `NamedTemporaryFile` will be returning a `IO[str]`. The appropriate return would be that instead of `TextIO`.